index.html

```diff
@@ -240,5 +240,34 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-container">
+         <div class="form-text">
+           <h1>Contact me</h1>
```

Comment:
- [ ] great job with the contact section. please kindly use the complete background image with the designs on it so it looks like the figma version
- [ ] please make your `contact me` heading bolder so it follows the figma design
your version
![Screenshot (250)](https://user-images.githubusercontent.com/65298048/179066401-473db0bb-844e-4403-a021-fe9141803982.png)
figma version
![Screenshot (249)](https://user-images.githubusercontent.com/65298048/179066437-896201f6-1c3f-4b9f-b56f-20b3b6d5ce58.png)